src/ukify/ukify.py

```diff
@@ -577,7 +579,10 @@ def check_inputs(opts: UkifyConfig) -> None:
 
         if isinstance(value, Path):
             # Open file to check that we can read it, or generate an exception
-            value.open().close()
+            if not value.is_dir():
+                value.open().close()
+            else:
+                value.iterdir()
```

Comment:
The comment is outdated now. Maybe something like this:
```
# Check that we can open the directory or file, or generate and exception
if value.is_dir():
   value.iterdir() 
else:
   value.open().close()
```
(We prefer to avoid negative conditions.)